OWNERS

```diff
@@ -2,10 +2,12 @@
 
 approvers:
  - russellb
- - maelk
  - kashifest
  - fmuyassarov
  - furkatgofurov7
 
 reviewers:
 - macaptain
```

Comment:
Nit: can you please add a space for consistency that we forgot while adding it?
```suggestion
 - macaptain
```

Reply:
done